### ISSUE TITLE
Double requests

### ISSUE THREAD

**Issue by newdeal30** [SQL-MisterMagoo/Blazor.PWA.MSBuild#25]
Hi MisterMagoo,

Thanks for your package. 
I know you stopped development on the package, but maybe you can answer me a question.
My server-side blazor pwa project uses identity. When i login to the page, the POST requests get send 2 times. 
When i look at the developer options from the browser i see, that 1 time the POST is send from the form, and one time from ServiceWorker.
Do you know that problem?

Thanks and regards
Michael

**Comment by SQL-MisterMagoo** [comment]
Are you sure it is being sent over the wire twice? 
Here is a sample from the network tab of one of mine 

![image](https://user-images.githubusercontent.com/11338430/84245079-7040d900-aafc-11ea-9449-e31c81bee96e.png)

You will see lots of cached responses (5th column - size - is greyed out) and two actual network requests (about 612 b each). 

On those two requests that made it to the network, you will see two entries in the list as the page requests the network data and the service worker intercepts and performs the actual call.

If you are seeing two actual network requests to the same resource, you might need to modify the service worker to just pass all requests to your identity provider without interruption.

**Comment by newdeal30** [comment]
Yes, it is send over the network twice. I get two requests to my controller. One from ServiceWorker.js, and one from the website itself. If i deregister the serviceworker and remove the includes from _Host.cshtml, things are working like they should.
Its also not only on the identity login. Same behavior when doing a GET/POST call from my swagger page.
Very strange...

PS: I am using ServerSide Blazor
![vs](https://user-images.githubusercontent.com/6523553/84359429-e743a380-abc8-11ea-85cc-d2317b75eec8.gif)

**Comment by newdeal30** [comment]
I have made a sample project, to see the behavoir. (https://github.com/newdeal30/BlazorPWAApp3)
To reproduce:
 Set a breakpoint in TestController/Get
 Navigate to https://localhost:5001/swagger
Try the method.
It should hit 2 times.

Thanks and regards 
Michael
